qa/tasks/scripts/ses_install_rgw_part2.sh

```diff
@@ -3,10 +3,10 @@ set -ex
 declare -a storage_minions=("$@")
 
 random_minion_fqdn=${storage_minions[0]}
-random_minion=`echo $random_minion_fqdn | cut -d . -f 1`
+random_minion=${random_minion_fqdn%.*}
 
 random_minion2_fqdn=${storage_minions[1]}
-random_minion2=`echo $random_minion2_fqdn | cut -d . -f 1`
+random_minion2=${random_minion2_fqdn%.*}
 
 salt $random_minion_fqdn service.status ceph-radosgw@rgw.${random_minion}.service 2>/dev/null
```

Comment:
Given the above statement about script parameters, isn't it better to use 'for' loop here to do same operations?

Reply:
all done in last 2 commits
c8dd46214535af339f2f0b3ddfbabf8c9048b5d6
0adbbfd81171ac22b16d6b1d155a2a1a9e2f00d7